about.html

```diff
@@ -4,15 +4,33 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="stylesheet.css">
-    <title>Document</title>
+    <script src="https://kit.fontawesome.com/8d3908a90c.js" crossorigin="anonymous"></script>
+    <title>Final Website</title>
 </head>
 <body>
-    <nav>
-        <ul>
-          <li><a href="index.html">Home</a></li>
-          <li><a href="about.html">About Us</a></li>
-          <li><a href="contact.html">Contact Us</a></li>
-        </ul>
-      </nav>
+  <nav>
+    <div class="navbar">
+      <a href="https://www.seasmoss.com/"><img class="logo" src="images.img/Original Logo.png" alt="Logo"></a>
```

Comment:
your image folder doesnt look like it has a good name. Should be `img` or `images` also you actual image should be all lowercase and no spaces. `original-logo.png`